sdlib/d/gc/region.d

```diff
@@ -91,6 +98,15 @@ public:
 		return (cast(RegionAllocator*) &this).computeAddressRangeImpl();
 	}
 
+	bool checkBlockThreshold() shared {
+		return nBlocks >= blockCollectThreshold;
+	}
+
+	void setBlockThreshold(int targetNum, int targetDen) shared {
+		blockCollectThreshold =
+			max(nBlocks * targetNum / targetDen, MinimumBlockCollectThreshold);
+	}
```

Comment:
I don't understand why this is in the region allocator. it doesn't seem very relevant to managing regions.

Reply:
We need to store a count of blocks, and a threshold of when to trigger the automatic allocation. I thought you had mentioned the region allocator as the correct place to store this?

Reply:
Perhaps the terminology can be changed to make it less related to collecting? I don't know.

Reply:
I have to insist on this. this is an indicator that the design is going in the wrong direction.

Reply:
So I still have these functions in here now, but they just forward to the value tracker. That is ok, right?